### DISCUSSION TITLE
Account creation

### DISCUSSION THREAD

**Discussion by Rohan7654** [EMACC99/mangadex#36] [Q&A] [answered]
Hey just wanted to cross check whether creating account is available or not cause i saw this line in readme file but couldn't find this function in apis.py file
>>> api.create_account(username = "your username", password = "your password", email = "youremail@example.com", ObjReturn = False)

**Comment by lasersPew** [comment, marked as answer]
> Hey just wanted to cross check whether creating account is available or not cause i saw this line in readme file but couldn't find this function in apis.py file
> 
> > > > api.create_account(username = "your username", password = "your password", email = "[youremail@example.com](mailto:youremail@example.com)", ObjReturn = False)

Check [#3](https://github.com/EMACC99/mangadex/issues/3#issuecomment-1869544034)
TLDR: "Never did, never will" the MD API support account creation
